tests/integration/targets/sudoers/tasks/main.yml

```
@@ -174,3 +212,5 @@
   ansible.builtin.assert:
     that:
       - edge_case_1 is failed
+      - edge_case_2 is not failed
+      - edge_case_3 is failed
```

You should also validate `edge_case_3.msg`, like this:
```suggestion
      - edge_case_3 is failed
      - edge_case_3.msg.starts_with('Failed to validate sudoers rule:')
```